.github/actions/build_extensions/action.yml

```diff
@@ -57,7 +57,7 @@ inputs:
   excel:
     description: 'Build Excel'
     default: 1
-  oote:
+  out_of_tree_ext:
     description: 'Build Out-Of-Tree Extensions'
     default: 0
```

Comment:
Since we are now building the OOT extensions also for windows, does it make sense to set the default to 1 and just always build the OOTE from extensions.csv in the CI? This will also enable them for the python specific extension binaries that are built in the `manylinux-extensions` job in `Python.yml` so we can use them from python.

I could also just submit a separate PR for this, just letmeknow!

Reply:
It does look unrelated to this PR